flocker/node/functional/test_docker.py

```diff
@@ -138,15 +138,6 @@ def start_container(self, unit_name,
 
         return d
 
-    def test_default_base_url(self):
-        """
-        ``DockerClient`` instantiated with a default base URL for a socket
-        connection has a client HTTP url after the connection is made.
-        """
-        client = DockerClient()
-        self.assertEqual(client._client.base_url,
-                         u'http+unix://var/run/docker.sock')
-
```

Comment:
To confirm - we only checked this because the testtools relied on it to check if it is possible to connect to Docker. This branch changes that check so the URL is not relied on anywhere. If that is correct, this is fine.